input/data/manure/refreshed_manure_management.json

```
@@ -1,7 +1,9 @@
 {
-    "anaerobic_digesters":[
+    "anaerobic_digester_configs":[
         {
             "name": "anaerobic_digester",
+            "processor_type": "anaerobic digester",
+            "digester_type": "anaerobic digester",
```

```suggestion
            "digester_type": "CSTR or continuous mix",
```
Just making this more specific. We only have one type of digester at the moment but this is the specific type of digester that the processor represents.

---

Would one or the other be better or should the full type really be `"CTSR or continuous mix"`?

---

I originally included them both because they're used sort of interchangeably. But I think just "continuous mix" would be sufficient so the type option is not confusing!